tools/hlslpm/README.md

```diff
@@ -0,0 +1,62 @@
+<!-- {% raw %} -->
+
+# hlslpm
+
+This tool helps manage github issues tracking the [LLVM HLSL
```

Comment:
Are we expecting to run this tool as an automated process to keep issues updated or will this be something we run manually?

Reply:
At the moment I'm running it manually and checking what it'll do. Will consider automating it later if it proves to be stable.  As an example, the first time I ran it I had to fix up a bunch of things because some changes were made to a milestone issue instead of a workstream issue and they'd have just been overwritten.